src/Microsoft.Framework.PackageManager/SourceControl/SourceCommand.cs

```diff
@@ -0,0 +1,253 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Framework.Runtime;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet;
+
+namespace Microsoft.Framework.PackageManager.SourceControl
+{
+    internal class SourceCommand
+    {
+        private const string SourcesFolderName = "sources";
+
+        private const string SourcesEnvironmentVariable = "DNX_SOURCES";
```

Comment:
:rant: I only wish we did not use the `CommonPrefix + ` pattern. It is impossible to find all occurrences of an environment variable across all the projects (managed and native) in the dnx if you are not aware of this class. I have been on my quest to understanding what is the benefit of the `CommonPrefix + ` pattern for some time now...